server/public/styles.css

```diff
@@ -0,0 +1,9 @@
+.card {
+  width: 30%;
+  height: 60vh;
+}
+
+.card > img {
+  height: 50%;
+  object-fit: contain;
+}
```

Comment:
Avoid styling 3rd-party classes!

Also, I do not recommend `vh` or percentage heights. Instead, pick a hard-coded `px` height.